CourseProject/src/main/java/com/neoflex/CourseProject/service/ValidationServiceImpl.java

```diff
@@ -0,0 +1,90 @@
+package com.neoflex.CourseProject.service;
+
+import com.neoflex.CourseProject.dto.LoanStatementRequestDto;
+import com.neoflex.CourseProject.exception.ValidationException;
+import com.neoflex.CourseProject.pojo.ValidationProperties;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.Period;
+
+@Service
+@Slf4j
+public class ValidationServiceImpl implements ValidationService {
+
+    private final ValidationProperties validationProperties;
+
+    @Autowired
+    public ValidationServiceImpl(ValidationProperties validationProperties){
+        this.validationProperties = validationProperties;
+    }
+
+    @Override
+    public void resultsOfValidation(LoanStatementRequestDto loanStatementRequestDto){
+        log.info(loanStatementRequestDto.toString());
+        areNamesValid(loanStatementRequestDto.getMiddleName());
+        isEmailValid(loanStatementRequestDto.getEmail());
+        areNamesValid(loanStatementRequestDto.getFirstName());
+        areNamesValid(loanStatementRequestDto.getLastName());
+        isPassportSeriesValid(loanStatementRequestDto.getPassportSeries());
+        isPassportNumberValid(loanStatementRequestDto.getPassportNumber());
+        isAmountValid(loanStatementRequestDto.getAmount());
+        isTermValid(loanStatementRequestDto.getTerm());
+        isAdult(loanStatementRequestDto.getBirthdate());
+        log.info("loanStatementRequestDto is valid");
+   }
+
+    private void isEmailValid(String email){
+       if (!email.matches(validationProperties.getEmailRegex())) {
+           log.info("validation exception email is not valid " + email);
```

Comment:
Показывал уже, как стоит изменить все логи:
```suggestion
           log.info("validation exception email is not valid: {}", email);
```
В таком случае будет отделение, а не сплошной текст в консоли

Reply:
исправлено